### PR TITLE
[1LP][RFR] Fix to broken test_db_restore

### PR DESCRIPTION
**Pull request by lcouzens**
PR to fix broken automation test, a future PR should follow to re-factor the test into fixtures and functions where possible, removing setup from the test itself.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 31eb25a2f20d70ed72233f8c35122734726f03b6:

`cfme/tests/test_db_restore.py`:
- :bangbang: Line 54:1: **E265** *block comment should start with '# '*
- :bangbang: Line 87:5: **E303** *too many blank lines (2)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by lcouzens** [comment]
Current 5.9 failure is a rhos template issue.

File: cfme/tests/test_db_restore.py [hunk 1]
```diff
@@ -1,131 +1,89 @@
...
     lambda: not store.current_appliance.is_downstream)
-def test_db_restore(request, soft_assert, virtualcenter_provider_crud, ec2_provider_crud,
-                    appliance):
-
-    appl1, appl2 = get_appliances()
-
-    def finalize():
-        appl1.destroy()
-        appl2.destroy()
-    request.addfinalizer(finalize)
```
Comment (quarckster) [inline cfme/tests/test_db_restore.py]: **QUESTION**
Why did you remove the finalizer?

Comment (lcouzens) [reply cfme/tests/test_db_restore.py]: This is handled by the appliance fixture now I thought? https://github.com/ManageIQ/integration_tests/blob/master/fixtures/appliance.py#L41
